### ISSUE TITLE
Vagrant 2.1.x is not compatible with plugin dependencies.

### ISSUE THREAD

**Issue by damontgomery** [palantirnet/the-vagrant#52]
We had issues when trying to install the vagrant plugins using the latest Vagrant version.

We could install them with 2.0.4 which can be installed with
```
brew cask install https://raw.githubusercontent.com/Homebrew/homebrew-cask/21b3b73976260b059becba363d5e33aaae3b1af8/Casks/vagrant.rb
```

The error produced with the new version is
```
WARNING: Vagrant has detected the `vagrant-triggers` plugin. This plugin conflicts
with the internal triggers implementation. Please uninstall the `vagrant-triggers`
plugin and run the command again if you wish to use the core trigger feature. To
uninstall the plugin, run the command shown below:

  vagrant plugin uninstall vagrant-triggers

Note that the community plugin `vagrant-triggers` and the core trigger feature
in Vagrant do not have compatible syntax.

To disable this warning, set the environment variable `VAGRANT_USE_VAGRANT_TRIGGERS`.
There was an error loading a Vagrantfile. The file being loaded
and the error message are shown below. This is usually caused by
a syntax error.

Path: /Users/---/projects/---/Vagrantfile
Line number: 6
Message: RuntimeError: vagrant-hostmanager plugin is not installed. Please install with: vagrant plugin install vagrant-hostmanager
```

**Comment by damontgomery** [comment]
This came about from troubleshooting issues with @Kbentham.

**Comment by becw** [comment]
Sweet, I'd be happy to remove this plugin from our dependencies. I hadn't seen that was added in [May as part of Vagrant 2.1.0](https://github.com/hashicorp/vagrant/blob/master/CHANGELOG.md#210-may-3-2018).

**Comment by becw** [comment]
Resolved by PR #54 -- this will be part of the next release.
